rest/src/db/CatapultDb.js

```diff
@@ -423,6 +432,57 @@ class CatapultDb {
 			})));
 	}
 
+	/**
+	 * It retrieves and adds the blocks information to the transactions' meta.
+	 *
+	 * The block information includes its timestamp and feeMultiplier
+	 *
+	 * @param {object[]} list the list without the added block information.
+	 * @returns {Promise<object[]>} this list with the added block information.
+	 */
+	async addBlockMetaToTransactionList(list) {
+		const isValidHeight = height => height && '0' !== height.toString();
```

Comment:
1. why do you need this check?
2. why are you comparing strings?

Reply:
1. For partial and unconfirmed transactions where the stored block is Long(0)
2. The stored height is actually Mongo's Long value, the unit tests stored just regular numbers. I have updated the unit tests so I can swap toString() to toInt()